### PR TITLE
Feature/progress

### PR DESCRIPTION
**Pull request by nicmoski**

### CODE REVIEW

File: client/classes/ApplicationState.js [hunk 1]
```diff
@@ -253,6 +267,49 @@ define(
...
+                var sec = avgStartTime - timeFromUpdate + adjustment;
+                
+                self.progress(sec);
+                var cdID = setInterval(countdown, 1000);
+                // keep track of this id so we can stop the countdown later
+                self.countdownid(cdID)
+            };
+
+            self._startCountdownAtRestart = ko.computed(function() {
+                // only show when admin is enabled
```
Comment (jalons) [inline client/classes/ApplicationState.js]: I like this, at least initially. Do we know if during startup people typically have admin enabled?

File: client/model/adminModel.js [hunk 2]
```diff
@@ -32,5 +33,9 @@ define(['knockout', 'jquery', './loginModel'],
                 });
         };
 
+        admin.toggleProgress = function() {
```
Comment (jalons) [inline client/model/adminModel.js]: Hooray!

**Comment by jalons** [comment]
Do we need to add documentation anywhere surrounding the offset (or the presence of this feature)?

File: client/model/adminModel.js (hunk 2 above)
Comment (nicmoski) [inline client/model/adminModel.js]: Initially it's going to be disabled until I can test how good the data is/how it affects performance/how people want it represented (sec, loading bar, etc). If al that goes well we can default enabled to True.

**Comment by nicmoski** [comment]
Probably a good idea. But I believe other members of the team have a backlog item to rethink the documentation piece so we can piggyback off of that.

**Pull request merged**